RStudio Testbed

leonardo-doyensec opened the pull request:
Testbed for RStudio Server

maoning commented:
@leonardo-doyensec there has been a folder structure change, could you move the testbed to rstudio/weak_credentials instead?

maoning commented:
I see the following message:

> tput: No value for $TERM and no -T specified
Settings by `-e USER=<new username>` is now deprecated and will be removed in the future.
Please do not use the USER environment variable.
tput: No value for $TERM and no -T specified

Do you know what is the recommended config? It would be nice to have both :)

leonardo-doyensec commented:
Hi @maoning ,
i've fixed what you have requested. Regarding the Dockerfile, i've removed the custom user since RStudio creates a user called `rstudio`. In order to run the plugin correctly, i've added the `rstudio` user to the list of common users (as described in the plugin's PR https://github.com/google/tsunami-security-scanner-plugins/pull/362)
